test-lib.sh

```diff
@@ -52,6 +52,42 @@ function ct_enable_cleanup() {
   trap ct_cleanup EXIT SIGINT
 }
 
+# ct_pull_image
+# -------------
+# Function pull an image before tests execution
+# Argument: image_name - string containing the public name of the image to pull
+# Function returns either 0 in case of pull was successful
+# Or the test suite exit with 1 in case of pull error
+function ct_pull_image() {
+  local image_name="$1"
+  local loops=10
```

Comment:
How about making it possible to set the amount of loops via function argument

Reply:
I will do it so, that If the parameter is not specified, then used default.